server/test/org/apache/cloudstack/network/lb/CertServiceTest.java

```diff
@@ -686,35 +689,35 @@ public void runDeleteSslCertBoundCert() throws NoSuchFieldException, IllegalAcce
         when(certService._entityMgr.findById(eq(LoadBalancerVO.class), anyLong())).thenReturn(new LoadBalancerVO());
 
         //creating the command
-        DeleteSslCertCmd deleteCmd = new DeleteSslCertCmdExtn();
-        Class<?> _class = deleteCmd.getClass().getSuperclass();
+        final DeleteSslCertCmd deleteCmd = new DeleteSslCertCmdExtn();
+        final Class<?> klazz = deleteCmd.getClass().getSuperclass();
 
-        Field certField = _class.getDeclaredField("id");
+        final Field certField = klazz.getDeclaredField("id");
         certField.setAccessible(true);
         certField.set(deleteCmd, certId);
 
         try {
             certService.deleteSslCert(deleteCmd);
             fail("Delete with a cert id bound to a lb should fail");
-        } catch (Exception e) {
+        } catch (final Exception e) {
             assertTrue(e.getMessage().contains("Certificate in use by a loadbalancer"));
         }
```

Comment:
Please consider replacing this `try` block with `expected` on the `@Test` annotation